null_safety_examples/misc/test/language_tour/browser_test.dart

```diff
@@ -13,7 +13,8 @@ void main() {
     document.body?.appendHtml(div);
     void reverseText(MouseEvent e) {}
 
-    // TODO(miquelbeltran) Maybe ! would be better here
+    // TODO(miquelbeltran) Maybe ! would be better here (#3295)
```

Comment:
Including the issue link here would be great as well. 

```suggestion
    // TODO(miquelbeltran) Maybe ! would be better here
    // (https://github.com/dart-lang/site-www/issues/3295)
```